src/Store.js

```diff
@@ -309,8 +307,8 @@ class Store {
     return snapshot
   }
 
-  async loadFromSnapshot (onProgressCallback) {
-    this.events.emit('load', this.address.toString())
+  async loadFromSnapshot () {
+    this.events.emit('load', this.address.toString()) //TODO: inconsistent params, line 207
```

Comment:
Can we fix the params?

Reply:
Not sure what direction to go with on the `load` and `replicated` events. In other words, should the `'load'` event emit the `address` and `heads` or just the `address`.

And with the `'replicated'` event, should it emit the `address` and number of entries replicated? or just the `address`.

Reply:
@haadcode should know.